tdrs-frontend/src/mirage.js

```diff
@@ -12,6 +13,23 @@ export default function startMirage(
       this.post('/reports/', () => {
         return 'Success'
       })
+      this.get('/reports/data-files/:year/:quarter/:section', () => {
```

Comment:
@riatzukiza just to confirm--will a download button only be visible if a file has been submitted? or will it be available upon upload? asking because #833 gave me the impression submission was required, but when I dropped a file in during testing (pre-submit), the download button appeared. Is this due to mirage? 
![416p1](https://user-images.githubusercontent.com/63075587/124616275-f0db2180-de43-11eb-9216-9f74ac62368b.PNG)